devtools/make_package.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Script to make a Caliopen package
+#
+
+BASEDIR="`pwd`/.."
+DISTDIR="${BASEDIR}/dist"
+
+
+function make_python_package() {
```

Comment:
I was thinking about a target directory argument in order to put the dist into where we want (or current working directory if none)
(eg. I want to package a temporary build for whatever ... integration test, quick fix of the front or backend to share with others, build a particular commit without overwriting other stable builts with same version etc.)